src/framework/components/anim/component.js

```diff
@@ -374,7 +357,27 @@ class AnimComponent extends Component {
     }
 
     /**
-     * @private
+     * @name AnimComponent#rootBone
+     * @type {Entity}
+     * @description The entity that this anim component should use as the root of the animation hierarchy
+     */
+    get rootBone() {
+        return this.data.rootBone;
+    }
+
+    set rootBone(value) {
+        if (typeof value === 'string') {
+            const entity = this.entity.root.findByGuid(value);
+            if (entity) this.data.rootBone = entity;

```

Comment:
maybe print a warning in debug mode that root bone cannot be found here if appropriate

Reply:
Done 👍